bedrock/base/templates/includes/protocol/navigation/menus/whoweare.html

```diff
@@ -73,15 +73,18 @@ <h4 class="c-menu-item-title">{{ ftl('navigation-v2-mozilla-blog') }}</h4>
               </a>
             </section>
           </li>
+
+          {% if ftl_has_messages('navigation-v2-impact', 'navigation-v2-find-out-how') %}
```

Comment:
I added this helper here as well so locales that haven't been translated for yet don't see this bit